docs/_includes/default.html

```diff
@@ -23,10 +23,9 @@
       <h1>
         <a href="/">
           <img
+            id="mocha-logo"
             src="/images/mocha-logo.svg"
-            alt="Mocha"
-            width="192"
-            height="192"
```

Comment:
Keep the dimensions; the HTML is rendered first so it's always better to have the dimensions here.